pkg/rancher-components/src/components/Form/Radio/RadioButton.vue

```diff
@@ -115,12 +116,21 @@ export default Vue.extend({
     /**
      * Emits the input event.
      */
-    clicked({ target }: { target?: HTMLElement }) {
-      if (this.isDisabled || target?.tagName === 'A') {
+    clicked(event: MouseEvent | KeyboardEvent) {
+      const target = event.target;
+
+      if (this.isDisabled || (target instanceof HTMLElement && target.tagName === 'A')) {
```

Comment:
Just for my own information, in what scenarios will target be something other than HTMLElement?

Reply:
We're really using `instanceof` to provide a more specific type check so that we can access `Event.target` without complaints from TypeScript.

Event.target is typed as `EventTarget | null`; the original code would produce compiler warnings

```
Property 'tagName' does not exist on type 'EventTarget'.ts
```